app/shared/src/errors/__tests__/GeneralError.test.js

```diff
@@ -2,11 +2,16 @@
 
 import * as React from 'react';
 import renderer from 'react-test-renderer';
+import { Translation } from '@kiwicom/mobile-localization';
 
 import GeneralError from '../GeneralError';
 
 it('renders as expected', () => {
   expect(
-    renderer.create(<GeneralError errorMessage="Error Message" />),
+    renderer.create(
+      <GeneralError
+        translation={<Translation passThrough="Error Message" />}
```

Comment:
I am not sure if this property renaming is a good idea. It should say what is this property about and not what type should you use...